CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.48.0 - 2022-02-10
+
+### Fixed
+
+- Fix [拍攝 capture 後未上傳完成 , 使用 "重整功能" 圖會直接被刪除](https://github.com/numbersprotocol/capture-lite/issues/918)
```

Comment:
For this one I think we can give it a short description in English and ref the issue by adding `#918` to it, see previous changelogs for example. We would like the changelog to have consistent format and English-only.

An example
```
### Fixed

- Phone verification code form duplicate paste issue. #839
```

Reply:
Ok! Just updated it!